supabase/migrations/20230426062304_create_wallet_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE wallets (
+    user_name character varying(255) NOT NULL PRIMARY KEY,
+    wallet_address character(42),
+    created_at text,
+    updated_at text,
+    multiplier text DEFAULT 1
+);
```

Comment:
It is not right. the migration file has only a multiplier field. 

Here are the steps to solve it.
1/ Remove 20230426..._create_wallet_table.sql first.
2/ Create a new migration with the name `add_multiplier_to_wallets`.
3/ Please copy/paste the following.
```sql
ALTER TABEL public.wallets
ADD COLUMN IF NOT EXISTS multiplier numeric DEFAULT 1
```

Reply:
> It is not right. the migration file has only a multiplier field.
> 
> Here are the steps to solve it. 1/ Remove 20230426..._create_wallet_table.sql first. 2/ Create a new migration with the name `add_multiplier_to_wallets`. 3/ Please copy/paste the following codebase.
> 
> ```sql
> ALTER TABLE public.wallets
> ADD COLUMN IF NOT EXISTS multiplier numeric DEFAULT 1
> ```

Done